apps/builder/app/shared/sync/sync-stores.ts

```diff
@@ -53,11 +53,13 @@ declare module "~/shared/pubsub" {
     sendStoreChanges: {
       // distinct source to avoid infinite loop
       source: SyncEventSource;
+      storeType: "client" | "server";
```

Comment:
alternative names for storeType: "destination", "location", "origin"

Reply:
Will get rid of it soon anyway.